resources/osl_mysql_test.rb

```diff
@@ -10,16 +10,22 @@
 property :server_password, String, default: 'osl_mysql_test'
 property :encoding, String, default: 'utf8mb4'
 property :collation, String, default: 'utf8mb4_unicode_ci'
+property :version, [String, nil]
```

Comment:
`nil` is implied and doesn't need to be set.
```suggestion
property :version, String
```